models/trail.js

```diff
@@ -19,6 +19,6 @@ const Trail = new Schema({
     'FCODE': String,
     'length_km': Number
   }
-}, {collection: 'newTrails'})
+}, {collection: 'trails'})
```

Comment:
Mostly because it has nothing to do with how the popup works, but I suppose it really isn't an issue.